src/platform/qt/library/LibraryTree.cpp

```diff
@@ -35,33 +34,42 @@ bool TreeWidgetItem::operator<(const QTreeWidgetItem& other) const {
 		QTreeWidgetItem::operator<(other));
 }
 
-LibraryTree::LibraryTree(QWidget* parent)
-	: QTreeWidget(parent)
-	, m_ui(new Ui::LibraryTree)
+LibraryTree::LibraryTree(LibraryController* parent)
+	: m_widget(new QTreeWidget(parent))
+	, m_controller(parent)
 {
-	m_ui->setupUi(this);
+	m_widget->setObjectName("LibraryTree");
+	m_widget->setSortingEnabled(true);
+
+	QTreeWidgetItem* header = new QTreeWidgetItem({
+		QApplication::translate("LibraryTree", "Name", nullptr),
+		QApplication::translate("LibraryTree", "Location", nullptr),
+		QApplication::translate("LibraryTree", "Platform", nullptr),
+		QApplication::translate("LibraryTree", "Size", nullptr),
+		QApplication::translate("LibraryTree", "CRC32", nullptr),
```

Comment:
Is this the correct way to do the equivalent of `QObject::tr` outside of a QObject?